docs/literate/src/files/adding_new_scalar_equations.jl

```diff
@@ -12,8 +12,8 @@
 # ## Basic setup
 using Trixi
 
-struct CubicEquation <: Trixi.AbstractEquations{1 #= number of spatial dimensions =#,
-                                                1 #= number of primary variables, i.e. scalar =#};
+struct CubicEquation <: Trixi.AbstractEquations{1, #= number of spatial dimensions =#
+                                                1} #= number of primary variables, i.e. scalar =#
```

Comment:
```suggestion
struct CubicEquation <: Trixi.AbstractEquations{1, # number of spatial dimensions
                                                1} # number of primary variables, i.e. scalar
```
With the comma moving in front of the comment, this is the more intuitive way to comment.